inference-engine/src/inference_engine/cnn_network_ngraph_impl.cpp

```diff
@@ -41,6 +41,8 @@ using ngraph::Function;
 static std::shared_ptr<ngraph::Function> copyFunction(const std::shared_ptr<const ngraph::Function>& func,
                                                       bool constFolding,
                                                       const std::map<std::string, std::vector<size_t>>& inputShapes) {
+    if (func == nullptr)
```

Comment:
why this can happen?

Reply:
In order to check that we can clone function and it is not nullptr